app/client/src/ce/sagas/PageSagas.tsx

```
@@ -536,15 +536,15 @@
         );
 
         if (actions && actions.length) {
-          yield put(setActionsToExecuteOnPageLoad(actions));
+          yield put(setActionsRunBehavior(actions));
         }
 
         const jsActions = actionUpdates.filter((d) =>
           d.hasOwnProperty("collectionId"),
         );
 
         if (jsActions && jsActions.length) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)